Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-snapshot.md

```diff
@@ -4,15 +4,8 @@ title: Snapshots
 
 ## Prerequisites
 
-- Rook officially supports v1 snapshots for Kubernetes v1.20+.
-
 - Install the snapshot controller and snapshot v1 CRD as required. More info can be found [here](https://github.com/kubernetes-csi/external-snapshotter/tree/v6.0.1#usage).
 
-!!! note
-    If only Alpha snapshots are available, enable snapshotter in `rook-ceph-operator-config` or helm chart `values.yaml`, change the external-snapshotter image to `registry.k8s.io/sig-storage/csi-snapshotter:v1.2.2` and refer to the [alpha snapshots documentation](https://github.com/rook/rook/blob/release-1.3/Documentation/ceph-csi-drivers.md#rbd-snapshots)
-
-    VolumeSnapshot betav1 is deprecated in Kubernetes 1.20+ and removed in 1.24.0. If you still require betav1 snapshots, change the external-snapshotter image to `registry.k8s.io/sig-storage/csi-snapshotter:v5.0.1` and refer to the [betav1 snapshots documentation](https://rook.github.io/docs/rook/v1.8/ceph-csi-snapshot.html#rbd-snapshots)
-
```

Comment:
Remove `## Upgrade Snapshot API` section below as its not relevant anymore

Reply:
suggestion above